data/deep missions.txt

```diff
@@ -0,0 +1,2306 @@
+				goto laura
+			
+			label electron
+			`	"Pierre is an older guy. He has been working on laser weaponry for the better part of three decades now. If you've ever seen an electron beam in action, you have him to thank for that. And a few others, of course, but one of his findings is really what brought the whole project together.`
+			
+			label laura
+			`	"Laura, on the other hand-"`
+			
+			label end
+			`	Before Garrison can finish speaking, the other three scientists return, helping each other carry a sealed metal box. "We're just going to set this in the back, if you don't mind," Pierre says. The scientists make themselves cozy in the bunk rooms and get ready to depart.`
+			`	A chirping from your ship's radio indicates you have received a message from the station. You open it and are not shocked at its contents:`
+			``
+			`		Your immediate departure is required.`
+			`					Carbuncle Station Docking Authority`
+			``
+			`	You signal to your crew and passengers that takeoff is imminent, and obey the wishes of the Lovelace Lab's research station.`
+				launch
+
+
+
+mission "Deep: Project Hawking: Prime"
+	landing
+	invisible
+	source "Prime"
+	to offer
+		has "Deep: Project Hawking: active"
+	to fail
+		has "Deep: Project Hawking: done"
+	npc kill
+		government "Independent"
+		personality waiting
+		ship "Marauder Raven (Engines)" "Krampus"
+	on offer
+		conversation
+			`As you approach the massive <planet> spaceport building, Pierre directs you to park on a landing pad on the third story close to the warehouse section. "We're picking up various alloys used in the creation of ships that we need samples of for testing," Pierre explains.`
+			`	The scientists speak with the spaceport authorities, who begin loading several tons of alloys into your cargo from an overhead platform. You sit with Pierre as you wait for them to finish.`
+			choice
+				`	"That's a lot of different alloys. What are they going to be used for?"`
+				`	"Why did we need to come all the way to Prime to get these alloys?"`
+					goto alloys
+			
+			`	Pierre cocks his head to the side as he looks at you. "We're going to destroy them. Short and simple. We need to see how different ship alloys hold up to the technology we're working on.`
+			
+			label alloys
+			`	"We could have gotten alloys from a shipyard local to the Deep, but Betelgeuse Shipyards gets regular shipments of alloys more commonly used by shipbuilders in the Dirt Belt, so it's really the closest place to get everything we need."`
+			choice
+				`	"Do you enjoy the work you do for the Deep?"`
+				`	"How long have you been working for the Deep?"`
+		
```

Comment:
I used guards for "Deep Security Guards" as a way to differentiate between the language used by the Navy and by Deep Security.

Reply:
Yeah but guards don't typically crew ships, and it doesn't seem like one would ever refer to ships' crewmen as guards.

Reply:
In the typical meaning of the word perhaps, but Deep Security has a culture of its own distinct from the Navy, so referring to Deep Security rank and file as guards instead of soldiers or troops is part of that. They see themselves as guardians of the Deep and very rarely venture outside of it (this string of missions being one such rare occasion).